### ISSUE TITLE
Toolbar disabled when entering text box with tab

### ISSUE THREAD

**Issue by disposition68** [fraywing/textAngular#830]
When tabbing through the elements of a form, the toolbar does not enable when entering textangular widget. Strangely, if I shift+tab in from the other direction, the toolbar enables.

I haven't been able to find it, but is there any part of the code suppressing this functionality that I can disable, or is this just a bug?

**Comment by pbassut** [comment]
Can you test it [here](http://textangular.com/)?
I couldn't reproduce this issue.

**Comment by disposition68** [comment]
So if you go to your demo, tab twice after entering the page. The focus is in the text entry area but the toolbar is not enabled. Clicking into the text entry area gets focus and enables the toolbar.

So I guess the general question is this: shouldn't focusing on the text entry area enable the toolbar, regardless of whether it's by click or by tab?

**Comment by pbassut** [comment]
Right. I'll mark it as a bug although I'm not sure if it is one. Even if it was a bug I wouldn't say it's a high priority one.
If you wanna make a PR to fix this I would be happy to merge.
